### DISCUSSION TITLE
면담내용 암호화 해야할까요?

### DISCUSSION THREAD

**Discussion by dongho108** [woowacourse-teams/2022-ternoko#380] [Q&A]
면담 내용에는 크루들의 민감한 내용들이 담길 수 있습니다.

이를 암호화해서 저희끼리는 확인을 못하도록 막고 코치에게 보여줄때는 디코딩을 해야할 것 같습니다.

그런데 이 방식을 적용해도 사전메일을 보내기 때문에 보낸 메일함에서 인터뷰 내용을 확인할 수 있습니다.

그러면 사전메일을 보내고 삭제해버리거나 사전메일 정책을 없애는 것도 방법일 것 같습니다.

여러분들의 의견이 필요해요

**Comment by HyeonbinSa** [comment]
우선, 코치 입장에서 사전 메일의 필요성이 가장 중요한 문제일 것 같습니다.
사전 메일이 필수적인 요소라면 해당 정책을 그대로 가져가되 데이터가 암호화 되지 않는다는 것을 명시해줘야할 것 같고,
사전 메일이 필수적인 요소가 아니라면 사전 메일 기능을 빼고 데이터를 암호화하여 저장하는 방식으로 사용되어야할 것 같습니다.

개인적인 입장으로는 민감한 내용이지만 전체적으로 공개되는 내용이 아닌 서비스 관리자에 한해 읽기가 "가능"한 부분이라고 생각합니다.
개인정보와 같이 필수 요소는 아니라고 생각되어 현행 유지에 한표입니다.

**Comment by her0807** [comment]
양방향 랜덤 암호화를 해야할 것 같은데, 현재 인적 리소스가 부족하기도 하고, 이번 스프린트 내에는 학습후 적용이 어려울 것 같아요 ㅠㅠㅠㅠ 

그래서 우선 면담 내용을 저장하는 부분으로 발생할 문제점을 인식 했다, 이 점을 정책적으로 민감한 내용은 담지 말아달라고 `notice`  하는 방법으로 풀어가고, 이후에 차차 디벨롭 하시죠! (애자일하게) ㅋㅋ ☺️
